util/internal/integrationtest/mapper/orm/src/main/java/org/hibernate/search/util/impl/integrationtest/mapper/orm/ReusableOrmSetupHolder.java

```diff
@@ -170,12 +180,12 @@ public static ReusableOrmSetupHolder withBackendMocks(BackendMock defaultBackend
 			allBackendMocks.add( defaultBackendMock );
 		}
 		allBackendMocks.addAll( namedBackendMocks.values() );
-		return new ReusableOrmSetupHolder( OrmSetupHelper.withBackendMocks( defaultBackendMock, namedBackendMocks ),
+		return new ReusableOrmSetupHolder( OrmSetupHelper.withBackendMocksLegacy( defaultBackendMock, namedBackendMocks ),
```

Comment:
Reminder this class is using legacy stuff and should still be converted.

By the way, I think you can take advantage of some JUnit 5 features, e.g. the ParameterResolver thing for setup methods expecting parameters of type `OrmSetupHelper.SetupContext` or `ReusableOrmSetupHolder.DataClearConfig`. The `@ReusableOrmSetupHolder.Setup` annotation might become completely unnecessary in that case.

Though of course, tests requiring bytecode enhancement might get in the way...